src/dft/backends/mklgpu/CMakeLists.txt

```diff
@@ -32,12 +32,17 @@ add_library(${LIB_OBJ} OBJECT
 )
 add_dependencies(onemkl_backend_libs_dft ${LIB_NAME})
 
-target_include_directories(${LIB_OBJ}
-  PUBLIC ${ONEMKL_INTERFACE_INCLUDE_DIRS}
-)
 target_include_directories(${LIB_NAME}
   PUBLIC ${ONEMKL_INTERFACE_INCLUDE_DIRS}
 )
+
+# Due to using the same file name for different headers in this library and in
+# MKL Intel library, we force the compiler to follow C++ Core Guideline SF.12
```

Comment:
```suggestion
# the Intel(R) oneAPI Math Kernel Library, we force the compiler to follow C++ Core Guideline SF.12
```

Reply:
Thank you @lhuot I updated the comment in [18bd3da](https://github.com/oneapi-src/oneMKL/pull/575/commits/18bd3da50a093f7826408b1c19384749ae93b52c)